### DISCUSSION TITLE
Can you recommend adding this instructions for better performance to pihole.conf?

### DISCUSSION THREAD

**Discussion by fazer1** [RPiList/specials#1296] [General]
Mike Kuketz from kuketz-blog.de is [just testing](https://www.kuketz-blog.de/pi-hole-ergaenzende-commandsbefehle-fuer-den-unbound-betrieb/) additional instructions for the `/etc/unbound/unbound.conf.d/pi-hole.conf`

Is this interesting for the RPiList project? Can you recommand to take over these instructions?

```
## Performance
# More cache memory, rrset=msg*2 | Default: 4m, 4m
msg-cache-size: 32m
rrset-cache-size: 64m
# Time  to  live [minimum|maximum] for RRsets and messages in the cache | Default: 0, 86400
cache-min-ttl: 3600
cache-max-ttl: 86400
# Serve old responses from cache with a TTL of 0 in the response without waiting for the actual resolution to finish | Default: no, 0
serve-expired: yes
serve-expired-ttl: 86400
# Fetch DNSKEYs earlier (DNSSEC): More cpu usage, less latency | Default: no
prefetch-key: yes
# Helps to reduce the query rate towards targets that get a very high nonexistent name lookup rate | Default: no
aggressive-nsec: yes

## Privacy | Default: no, no
hide-identity: yes
hide-version: yes
```

**Comment by LizenzFass78851** [comment]
Try opening an issure with reference to this discussion.  Maybe @RPiList will then respond if necessary.

**Comment by RPiList** [comment]
Sorry for responding quite late.

I read the article on kuketz-blog.de, but I didn't get the "why" should the changes be made. He says the default settings are conservative... but didn't mention any advantages his settings achive.


Greetings

**Comment by RPiList** [comment]
okay, scratch that. Now I saw the comments in the config file...
